apollo-router-core/src/plugins/mod.rs

```diff
@@ -1,4 +1,5 @@
 mod forbid_mutations;
 mod headers;
 mod include_subgraph_errors;
+pub mod serde_utils;
```

Comment:
Does this need to be pub?

Reply:
It's used in multiple plugins whenever we need to deserialize `http::Header*`. In particular it's used now in `apolo-router` for the otel plugin. We could consider a general top level serde module as a separate followup ticket.